app/mailers/user_mailer.rb

```diff
@@ -206,11 +198,11 @@ def wiki_content_added(user, wiki_content)
 
     with_locale_for(user) do
       subject = "[#{@wiki_content.project.name}] #{t(:mail_subject_wiki_content_added, id: @wiki_content.page.pretty_title)}"
-      mail to: user.mail, subject: subject
+      mail_for_author author, to: user.mail, subject: subject
     end
   end
 
-  def wiki_content_updated(user, wiki_content)
+  def wiki_content_updated(user, wiki_content, author)
```

Comment:
Assignment Branch Condition size for wiki_content_updated is too high. [20.22/15]